src/content/academic-grants-2022.mdx

```diff
@@ -0,0 +1,205 @@
+---
+metaTitle: Academic Grants Round 2022
+metaDescription: The Ethereum Foundation is sponsoring a wave of grants to support Ethereum-related academic work. Find all the details you need to apply here.
+metaImage: /images/academic-grants-hero.png
+isClosed: true
+---
+
+**We're calling all those interested in expanding academic knowledge throughout the Ethereum ecosystem!**
+
+The Ethereum community has grown immensely as developers, artists, companies, and even governments work together to make a positive impact with this technology.
+
+With this grants round, the Ethereum Foundation welcomes more academics to the table to push forward formal academic research in related domains. Such domains include theoretical and applied cryptography and mathematics, zero-knowledge proofs, economics, computation, cybersecurity, protocol and consensus mechanisms, formal verification, P2P networking, and hardware, just to mention a few areas.
+
+The Ethereum Foundation is excited to expand Ethereum's reach across more of academia through this grants round.
```

Comment:
This used to be in a description section

```
 <section id='description' ref={ref}>
    <PageSubheading mb={8}>
      We&apos;re calling all those interested in expanding academic knowledge
      throughout the Ethereum ecosystem!
    </PageSubheading>

    <PageText mb={6}>
      The Ethereum community has grown immensely as developers, artists, companies,
      and even governments work together to make a positive impact with this
      technology.
    </PageText>

    <PageText mb={6}>
      With this grants round, the Ethereum Foundation welcomes more academics to the
      table to push forward formal academic research in related domains. Such domains
      include theoretical and applied cryptography and mathematics, zero-knowledge
      proofs, economics, computation, cybersecurity, protocol and consensus
      mechanisms, formal verification, P2P networking, and hardware, just to mention a
      few areas.
    </PageText>

    <PageText>
      The Ethereum Foundation is excited to expand Ethereum&apos;s reach across more
      of academia through this grants round.
    </PageText>
  </section>
  ```
  Which resulted in a summary item in the sidenav
  
  
![Screenshot 2024-03-16 at 11 58 27 AM](https://github.com/ethereum/esp-website/assets/15589226/fbecaa33-c23d-4329-a87c-b8c2f4d2ee05)

Just wondering if this is intentionally not there anymore, or maybe its missing.

Reply:
Good observation. The thing is that this is not something I have added support for, because now every heading will appear in the sidebar. So I made that heading bold to avoid having it in the sidebar.

Then, in the case of the Summary section, this is a special case that I also didn't add support for. It is special because it has a hidden heading in the content. With this implementation we would have to add a heading like `### Summary` to appear in the sidebar.

Let me see if we can do something about that.